### PR TITLE
Peer code review PR

### PR DESCRIPTION
**Pull request by benbourgon**
- Was intended to pull the initial timer values from Firebase, but it's currently not getting the info in time despite using useEffect
- timer values hard coded instead
- tomato values do not yet update after completion
- can't yet set active task

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+    // store the database info
+    const database = getDatabase(firebase);
+
+    // the database reference point
+    const dbReference = ref(database);
+
+    // listen for changes in the database
+    // and console log the values
+    onValue(dbReference, (dbResponse) => {
+      console.log(dbResponse.val());
```
Comment (FjPierre) [inline src/App.js]: remember to remove your console logs.

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <ul>
+        {/* {toDoList.map((toDoItem) => {
```
Comment (FjPierre) [inline src/App.js]: Also remove the extraneous code/comments

File: src/App.js [hunk 3]
```diff
@@ -27,11 +27,12 @@ const App = () => {
 
   useEffect(() => {
     // listen for changes in the database
-    onValue(dbReference, (dbResponse) => {
+    onValue(dbTasksReference, (dbResponse) => {
       // Create a variable to store our new to-do list when something changes
       const newToDoList = [];
       // store response from Firebase
       const data = dbResponse.val();
+      console.log(data)
```
Comment (FjPierre) [inline src/App.js]: remember to remove the console log

File: src/App.js [hunk 4]
```diff
@@ -94,11 +96,11 @@ const App = () => {
           tomatoesInputValue={tomatoesInputValue}
         />
         <Instructions />
-        <DisplayToDoList 
+        {/* <DisplayToDoList
```
Comment (FjPierre) [inline src/App.js]: and to remove extraneous code

File: src/App.js [hunk 5]
```diff
@@ -44,39 +46,59 @@ const App = () => {
...
+    // store the database and the reference point for this specific to-do item
+    // const database =  getDatabase(firebase);
+    // const dbToDoReference = ref(database, `/${toDoKey}`)
+    // change the state of the specified to-do item
   }
   return (
     <div className="App">
       <Header />
       <main>
         <ToDoInputForm
```
Comment (FjPierre) [inline src/App.js]: I like that you used the form to add your inputs 🙌🏾

File: src/Components/DisplayToDoList.js [hunk 6]
```diff
@@ -18,11 +18,11 @@ const DisplayToDoList = (props) => {
                                 <input
```
Comment (FjPierre) [inline src/Components/DisplayToDoList.js:19]: Like Esther said, remember to add a form for your inputs

**Review by FjPierre — APPROVED** [review]
Great code overall, just remember to remove console logs and extraneous comments/code. Great job Ben

File: src/App.js (hunk 1 above)
Comment (benbourgon) [reply src/App.js]: Thanks for catching that!

File: src/Components/DisplayToDoList.js (hunk 6 above)
Comment (benbourgon) [reply src/Components/DisplayToDoList.js:19]: Now added!

File: src/App.js (hunk 4 above)
Comment (benbourgon) [reply src/App.js]: Done, thanks!

File: src/App.js (hunk 3 above)
Comment (benbourgon) [reply src/App.js]: Good catch!

**Pull request merged**